guides/source/action_controller_overview.md

```diff
@@ -679,14 +679,14 @@ end
 
 You may notice in the above code that we're using `render xml: @users`, not `render xml: @users.to_xml`. If the object is not a String, then Rails will automatically invoke `to_xml` for us.
 
-Filters
+Action callbacks
 -------
 
-Filters are methods that are run "before", "after" or "around" a controller action.
+Action callbacks are methods that are run "before", "after" or "around" a controller action.
 
-Filters are inherited, so if you set a filter on `ApplicationController`, it will be run on every controller in your application.
+Action callbacks are inherited, so if you set an action callback on `ApplicationController`, it will be run on every controller in your application.
 
-"before" filters may halt the request cycle. A common "before" filter is one which requires that a user is logged in for an action to be run. You can define the filter method this way:
+"before" action callbacks may halt the request cycle. A common "before" action callback is one which requires that a user is logged in for an action to be run. You can define the action callback method this way:
```

Comment:
For here and each below, I think '`foo_action` callback[s]' will read better than '"foo" action callback[s]'. It was harder before, because there was no accompanying noun, and you can't really pluralize "`before_filter`s"; now that we have the "callback" there, let's use it.

For the last sentence, "callback method" seems sufficient.